client/src/js/screens/pages/Home/index.js

```diff
@@ -57,18 +134,18 @@ export default class Home extends React.Component {
         {/* End of the jumbotron area */}
         {/* Top rated recipes section */}
         <div className="container-fluid px-5 my-5">
-          <h1 className="display-4 text-center my-5 wow fadeInDown" style={{padding: '30px 0px'}}>
+          <h1 className="display-5 text-center my-5 wow fadeInDown" style={{padding: '30px 0px'}}>
             <i className="ion ion-star mr-3" />
-            Top Rated Recipes</h1>
+            Lastest Recipes</h1>
           <br />
           <div className="card-deck wow fadeIn">
-                       
+              {latestRecipes}
           </div>
         </div>
         {/* End of top rated recipes section */}
-        {/* Top rated chefs section */}
+        {/* Top rated chefs section 
         <div className="container px-5 my-5">
-          <h1 className="display-4 text-center my-5 wow fadeInDown" style={{padding: '30px 0px'}}>
+          <h1 className="display-5 text-center my-5 wow fadeInDown" style={{padding: '30px 0px'}}>
```

Comment:
According to airbnb style guides, the only comment in your code should be prefixed with `FIXME` or `TODO` this is a large chunk of code that should be deleted instead of you commenting out. 

Please refer to this link for more details 
https://github.com/airbnb/javascript#comments